### PR TITLE
[normal][3/6/2022]:Footer

### PR DESCRIPTION
**Pull request by bisnhu-gurung123**
demo_footer

### CODE REVIEW

File: demo-app/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <mat-divider class="footer__line"></mat-divider>
+    </div>
+  </div>
+  <div fxLayout="row" fxLayoutAlign="center center" fxLayoutGap="300px">
+    <div class="footer__description">
+      <p>The Swiss Ski to intiate the mass <br>
+        campaign for all its upcoming products<br>
+        The Swiss Ski to intiate the mass</p>
+      <p>
+        <button mat-raised-button>demo button</button>
```
Comment (Dhanvr) [inline demo-app/src/app/footer/footer.component.html:14]: i think p is not necessary

File: demo-app/src/app/footer/footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.container {
+  width: 1680px;
+  margin: auto;
```
Comment (Dhanvr) [inline demo-app/src/app/footer/footer.component.scss:3]: make this container global

File: demo-app/src/app/footer/footer.component.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  }
+
+  &__terms {
+    color: var(--color);
+    font-weight:bold;
+  }
+
+  &__icon {
+    color: var(--color);
```
Comment (Dhanvr) [inline demo-app/src/app/footer/footer.component.scss:34]: give variable name meaningful

File: demo-app/src/assets/stylesheet/_mixin.scss [hunk 4]
```diff
@@ -9,3 +9,4 @@
 @include boxModel((4, 8, 16, 32, 36), 'mr', 'margin-right');
```
Comment (Dhanvr) [inline demo-app/src/assets/stylesheet/_mixin.scss:9]: make sure you are understanding mixin

**Review by Dhanvr — APPROVED** [review]

File: demo-app/src/app/footer/footer.component.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<div class="footer container my-32" fxLayout="column" fxLayoutAlign="start center" fxLayoutGap="95px">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/footer/footer.component.html:1]: demo_footer

**Pull request merged**